tests/garage/torch/q_functions/test_discrete_cnn_q_function.py

```diff
@@ -0,0 +1,75 @@
+import pickle
+
+import pytest
+import torch
+from torch import nn
+
+from garage.envs import GymEnv
+from garage.torch.q_functions import DiscreteCNNQFunction
+
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestDiscreteCNNQFunction:
```

Comment:
do these need xunit style? we've been trying to encourage new tests use pytest style.